### PR TITLE
Don't log timing stats for long-lived neo requests

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**
Blocked on [swarm v4.4.0](https://github.com/sociomantic-tsunami/swarm/milestone/25).

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
This relates only to neo requests, so shouldn't be in this milestone.

**Comment by gavin-norman-sociomantic** [comment]
Blocked on #40.

**Comment by gavin-norman-sociomantic** [comment]
Rebased on #40.

**Comment by gavin-norman-sociomantic** [comment]
#40 merged. Unblocked.

**Comment by nemanja-boric-sociomantic** [comment]
Hm, this targets the wrong branch?

**Comment by gavin-norman-sociomantic** [comment]
Oops, fixed.

File: src/dhtnode/node/RequestHandlers.d [hunk 1]
```diff
@@ -34,10 +34,16 @@ public ConnectionHandler.CmdHandlers requests;
...
 static this ( )
 {
-    requests.add(RequestCode.GetHashRange, "GetHashRange", &GetHashRange.handle);
-    requests.add(RequestCode.Put, "Put", &Put.handle);
-    requests.add(RequestCode.Get, "Get", &Get.handle);
-    requests.add(RequestCode.Mirror, "Mirror", &Mirror.handle);
-    requests.add(RequestCode.GetAll, "GetAll", &GetAll.handle);
-    requests.add(RequestCode.GetChannels, "GetChannels", &GetChannels.handle);
+    requests.add(
+        RequestCode.GetHashRange, "GetHashRange", &GetHashRange.handle, false);
```
Comment (nemanja-boric-sociomantic) [inline src/dhtnode/node/RequestHandlers.d]: GetHashRange is now long-lived request, right?

Comment (gavin-norman-sociomantic) [reply src/dhtnode/node/RequestHandlers.d]: Yeah, it's active for the whole lifetime of the connection.

**Pull request merged**